### PR TITLE
feat(web): lpaq review complete path

### PR DESCRIPTION
**Pull request by kandcdev**
## Describe your changes
Ability to review LPAQ with "Complete" review outcome

## Issue ticket number and link
https://pins-ds.atlassian.net/browse/BOAT-133

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/appeals/appeal-details/lpa-questionaire-review/lpa-questionnaire-review.controller.js [hunk 1]
```diff
@@ -23,3 +28,76 @@ export async function viewLpaQuestionnaire(request, response) {
...
+			formattedSections.push(generateSummaryList(section.header, section.rows));
+		}
+
+		return response.render('appeals/appeal/lpa-questionnaire-view.njk', {
+			summaryList: { formattedSections },
+			errors
+		});
+	}
+	// If it succeeds validation
+	try {
```
Comment (james-hill-kc) [inline apps/web/src/server/appeals/appeal-details/lpa-questionaire-review/lpa-questionnaire-review.controller.js:53]: Is the try/catch necessary here?

File: apps/web/src/server/appeals/appeal-details/lpa-questionaire-review/lpa-questionnaire-review.controller.js [hunk 2]
```diff
@@ -23,3 +28,76 @@ export async function viewLpaQuestionnaire(request, response) {
...
+	/*
+	TODO:
+	- uncomment when generic "complete" page is done
+	- provide correct path to its .njk
+	- remove logger.debug
+	- write test for the review complete page
+	*/
+	logger.debug(`LPA questionnaire review complete page: ${appealId}, ${lpaQId}`);
+
+	response.render('appeals/appeal/generic-complete-page.njk', {
```
Comment (james-hill-kc) [inline apps/web/src/server/appeals/appeal-details/lpa-questionaire-review/lpa-questionnaire-review.controller.js:99]: Does this throw an error currently? The referenced page template doesn't exist, which is clearly intentional, but just wondering what happens when this render is called?

**Review by james-hill-kc — CHANGES_REQUESTED** [review]
Looking good! I've left a few comments

File: apps/web/src/server/appeals/appeal-details/lpa-questionaire-review/lpa-questionnaire-review.controller.js (hunk 2 above)
Comment (kandcdev) [reply apps/web/src/server/appeals/appeal-details/lpa-questionaire-review/lpa-questionnaire-review.controller.js:99]: Yes, it does throw error now

File: apps/web/src/server/appeals/appeal-details/lpa-questionaire-review/lpa-questionnaire-review.controller.js [hunk 3]
```diff
@@ -23,3 +28,76 @@ export async function viewLpaQuestionnaire(request, response) {
...
 }
+
+/** @type {import('@pins/express').RequestHandler<Response>} */
+export const postLpaQuestionnaire = async (
+	{ params: { appealId, lpaQId }, body, errors },
+	response
+) => {
+	// If it fails validations
+	if (errors) {
+		const lpaQuestionnaireResponse = await getLpaQuestionnaireFromId(appealId, lpaQId);
```
[the remark this answers is not in the document]
Comment (kandcdev) [reply apps/web/src/server/appeals/appeal-details/lpa-questionaire-review/lpa-questionnaire-review.controller.js]: done

**Review by james-hill-kc — APPROVED** [review]

**Pull request merged**